pkg/skaffold/render/renderer/renderer.go

```diff
@@ -45,26 +45,21 @@ const (
 )
 
 type Renderer interface {
-	Render(context.Context, io.Writer, []graph.Artifact) error
+	Render(context.Context, io.Writer, []graph.Artifact, bool, string) error
```

Comment:
can you use named parameters to explain what the `bool` and `string` represent? Also `SkaffoldRenderer` seemed like the only implementation of this interface and it also doesn't use those two arguments. If so can we remove them?

Reply:
alternatively I'd be fine with a paragraph comment explaining the parameters and the behavior of the function, but +1 to removing the args if they're unused and aren't planned to be

Reply:
It's the `offline` and `outputPath` fields which are defined [here](https://github.com/GoogleContainerTools/skaffold/blob/24d332cf35398c503276af6384faed18b4fbf6c8/pkg/skaffold/runner/runner.go#L51). This function will replace the deployers' `Render` function later on. Changed to named parameters (the actually logic will be implemented once hook up with the runner pipeline).

Reply:
might be better managed with a `RenderOptions` struct parameter instead.

```go
type RenderOptions struct {
    RunOffline bool
    OutputPath string
}
``

Reply:
This Renderer will inherit the Runner interface so the params has to be as defined https://github.com/GoogleContainerTools/skaffold/blob/24d332cf35398c503276af6384faed18b4fbf6c8/pkg/skaffold/runner/runner.go#L51. 

May be clearer once you see the next PR 😸